pcmanfm-desktop.c uses ceil(3) so must be linked to libm.

--- src/Makefile.am.orig	2012-01-16 17:03:19.091356701 +0100
+++ src/Makefile.am	2012-01-16 17:03:34.625980985 +0100
@@ -48,6 +48,7 @@ pcmanfm_LDADD = \
 	$(GMODULE_LIBS) \
 	$(FM_LIBS) \
 	$(MENU_CACHE_LIBS) \
+	-lm \
 	$(NULL)
 
 noinst_PROGRAMS=xml-purge
